src/Package/Util/Header/Version.php

```diff
@@ -90,14 +90,19 @@ public function fileHasVersionMacro($fname)
     public function getVersionFromHeader()
     {
         $headers = glob($this->package->getSourceDir().DIRECTORY_SEPARATOR.'*.h');
-        $pat = ',define\s+'.$this->macroName.'\s+"(.*)",i';
+
+        # Match versions surrounded by quotes and versions without quotes
+        $versionMatcher = '(".*"|.*\b)';
+        $pat = ',define\s+'.preg_quote($this->macroName, ',').'\s+'.$versionMatcher.',i';
```

Comment:
To try and make the regex easier to read. By separating the specific part that matches a version, the context around it is clearer, it's obvious where the version match begins/ends and what is part of the general line matching